README.md

```diff
@@ -1,7 +1,13 @@
 [![Starware](https://img.shields.io/badge/Starware-⭐-black?labelColor=f9b00d)](https://github.com/zepfietje/starware)
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
```

Comment:
Why is this duplicated?

Reply:
fixed. [changes](https://github.com/dscnitrourkela/project-udon/pull/17/commits/25b1a1a1ed993d4a08e4fba0e90c16de2898e65d)